pkg/clicmd/param_delete_test.go

```diff
@@ -45,6 +45,12 @@ func Test_paramDeleteCmd(t *testing.T) {
 				actions.OptionEnvName: "default",
 			},
 		},
+		{
```

Comment:
This won't be an error since the code in `clicmd` only collects arguments and doesn't know anything about the logic of applying them. You should add a test to `pkg/actions/param_delete_test.go` instead.